terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/logging/LoggingTest.java

```diff
@@ -50,7 +50,7 @@ public void test01_01_createDefaultXTrackMDC() {
 
         // cut x-Track MDC
         String targetMdc = driver.findElement(By.id("xTrackMDC")).getText();
-        String footerMdc = driver.findElement(By.cssSelector("p")).getText();
+        String footerMdc = driver.findElement(By.id("xtrack")).getText();

```

Comment:
@kazuki43zoo @yoshikawaa 
I made new issue #609 (Some methods should define at WebDriverOperations class?).